test/ci_tests/header_check.py

```diff
@@ -124,7 +124,9 @@ def get_committed_files():
     files = []
     for line in result.stdout.splitlines():
         status, *file_path = line.split("\t")
-        if status != "D":
+        if status.startswith("R"):
+            files.append(file_path[1])
+        elif status != "D":
```

Comment:
This fixes a pre-commit bug.  It could go in a separate PR but it's only 4 lines and needed here.  It is when files are renamed in git, the result of the status check only needs to do the header check on the destination file.